src/style.css

```
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Heavy.ttf");
+
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 300;
  color: #3d4e61;
  src: url("./fonts/Avenir-Book.ttf");
}

@font-face {
  font-family: "Avenir";
  font-weight: 800;
  color: #3d4e61;
  src: url("./fonts/Avenir-Heavy.ttf");

}
```
use the same font name but a different font-weight